hw5.py

```diff
@@ -0,0 +1,21 @@
+str1 = input('Enter text: ')
+for s in str1:
+    if s.isnumeric() == True:

```

Comment:
You can skip `== True` part and use the result of boolean functions here like `if s.isnumeric():`